<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Room_wait</class>
 <widget class="QDialog" name="Room_wait">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>等待中</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Wait.qrc">
    <normaloff>:/new/prefix1/Image/Icon/图标.jpg</normaloff>:/new/prefix1/Image/Icon/图标.jpg</iconset>
  </property>
  <widget class="QPushButton" name="start_Button">
   <property name="geometry">
    <rect>
     <x>107</x>
     <y>320</y>
     <width>155</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-image: url(:/new/prefix1/Image/Button/150.50.jpg);</string>
   </property>
   <property name="text">
    <string>开始游戏</string>
   </property>
  </widget>
  <widget class="QLabel" name="waitFor_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>230</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文琥珀</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>等待对方加入房间</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="backgroun_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>370</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/Image/Background/wait_副本.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="player1_label">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>210</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/Images/Hint/提示_小.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="player2_label">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>60</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/Images/Hint/提示_小.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="roomName_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 85, 127);</string>
   </property>
   <property name="text">
    <string>房间名</string>
   </property>
  </widget>
  <widget class="QLabel" name="roomNum_label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 85, 127);</string>
   </property>
   <property name="text">
    <string>房间号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(254, 255, 175);
background-color: rgb(219, 197, 158);</string>
   </property>
   <property name="text">
    <string>离开房间</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="roomType_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 85, 127);</string>
   </property>
   <property name="text">
    <string>房间类型</string>
   </property>
  </widget>
  <widget class="QLabel" name="my_name_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>290</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="op_name_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 85, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>backgroun_label</zorder>
  <zorder>start_Button</zorder>
  <zorder>waitFor_label</zorder>
  <zorder>player1_label</zorder>
  <zorder>player2_label</zorder>
  <zorder>roomName_label</zorder>
  <zorder>roomNum_label</zorder>
  <zorder>back_button</zorder>
  <zorder>roomType_label</zorder>
  <zorder>my_name_label</zorder>
  <zorder>op_name_label</zorder>
 </widget>
 <resources>
  <include location="Wait.qrc"/>
 </resources>
 <connections/>
</ui>
